### DISCUSSION TITLE
Report quantifier instantiations according to `:qid`

### DISCUSSION THREAD

**Discussion by xldenis** [cvc5/cvc5#11160] [Q&A]
Is it possible to get `cvc5` to report quantifier instantiations like `z3` where we can provide a user-supplied `:qid` attribute? The `-o inst` command provides the whole quantifier term, but for my purposes I'd rather have just the name.

**Comment by ajreynol** [comment]
The `-o inst` output should take into account `:qid` by default (I can confirm with some examples it does). 

Although one possible complication is if the quantified formula is destructively modified.

Do you have an example where cvc5 is not reporting instantiations based on `:qid`?
